src/xdocs/config_coding.xml

```diff
@@ -3324,6 +3324,22 @@ class TestHashCode {
         <source>
 &lt;module name=&quot;MissingCtor&quot;/&gt;
         </source>
+        <p>Example:</p>
+        <source>
+class ExampleOk { // OK
+    private int a;
```

Comment:
nested elements indentation is 4, but it should be consistent with javadoc and be 2

Reply:
Done

Reply:
Done